packages/react-query/src/useSuspenseQueries.ts

```diff
@@ -1,9 +1,10 @@
+// TODO: remove this eslint
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
After removing enabled option in @suspensive/react-query, I will remove these comments